packages/react-component-library/src/components/TabSet/TabContent.tsx

```diff
@@ -18,9 +18,9 @@ export const TabContent: React.FC<TabContentProps> = ({
   <StyledContent
     $isActive={isActive}
     role="tabpanel"
-    aria-labelledby={tabId}
+    aria-labelledby={`tab-button-${tabId}`}
```

Comment:
I noticed the `aria-labelledby` here appeared to be incorrect as it was the same as the `id` for this element. 

I've changed it to be the `id` of the `tab` (matching the GOV.UK Design System).